contents/data/sql/SQL.md

```diff
@@ -0,0 +1,151 @@
+
+    _Figure 2. MySQL (an RDBMS) can be used in the MySQL Workbench GUI application_ <sup>[source](https://www.mysql.com/products/workbench/)</sup>  
+
+    <img src="MySQL_CLI.png" alt="Running MySQL from the command line" width="60%">  
+
+    _Figure 3. Logging in to the MySQL RDBMS and viewing tables from the command line_ <sup>[source](https://www.researchgate.net/figure/The-MySQL-command-line-tool_fig5_328093393)</sup>  
+    </center>
+
+- **APIs**: You can use APIs in programming languages or independent ones like [SQL API](https://carto.com/developers/sql-api/) (shown in Figure 4) to interact with databases. For example, the [SQLite3](https://www.pythoncentral.io/introduction-to-sqlite-in-python/) library can be used to interact with a database through Python.
+
+    <center>
+    <img src="SQL_API.png" alt="Using SQL API to execute queries" width="80%">  
+
+    _Figure 4. An example of how to use CARTO's SQL API, which uses a PostgreSQL database_ <sup>[source](https://carto.com/developers/sql-api/)</sup>  
+    </center>
+
+---
+
+## Why learn SQL?
+
+The main reason why you should learn SQL relates to the widespread adoption of SQL. As SQL is not limited to any particular operating system or programming language, it is **widely used** in the RDBMSs used by a lot of software applications (web, mobile etc.). Hence, in order to utilize almost any relational database or RDBMS available today, you need to know the SQL language.
+
+## Disadvantage
+
+The biggest issue in SQL is with regards to the syntax and features.
+Although SQL databases use established <tooltip content="American National Standard Institutes">ANSI</tooltip> & <tooltip content="International Organization for Standardization">ISO</tooltip> standards, some RDBMSs %%(like PostgreSQL, for example)%% add proprietary extensions to standard SQL.
+Due to this, the available feature set can vary according to what you're using.
+Furthermore, there may be minor changes in the syntaxes across RDBMSs (case-sensitivity, for example).
+These factors can make SQL confusing and frustrating to use when switching across RDBMSs, since you would have to change your SQL code.
+
+---
+
+## How to get started with SQL?
+
+As you saw in the example in the previous section, SQL is not really that complex. It is easy to learn, even for beginners who do not have any prior experience with databases.
+
+To get started with SQL, you can either just use an online SQL playground or install a RDBMS. There are many free RDBMSs available like [PostgreSQL](https://www.postgresql.org/) and [MySQL](https://dev.mysql.com/downloads/).
+However, for the sole purpose of learning SQL, our suggestion is to use online editors like [DB Fiddle](https://www.db-fiddle.com/) %%(supports MySQL and SQLite)%% or [SqliteOnline](https://sqliteonline.com/) %%(supports SQLite)%% since they allow you to start learning without the hassle of setting up anything.
```

Comment:
>However, for the sole purpose of learning SQL, our suggestion is to use online editors

I wouldn't go as far as to _recommend_ it. Something like this? `At the early stages of learning SQL, you can even use online editors ...`